modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -974,6 +975,24 @@ public void testCfgGap() throws InterruptedException {
         checkTableWithData(newNode, "t2");
     }
 
+    /**
+     * The test for updating cluster configuration with the default value.
+     * Check that new nodes will be able to synchronize the local cluster configuration.
+     */
+    @Test
+    public void updateClusterCfgWithDefaultValue() {
+        IgniteImpl ignite = startNode(0);
+
+        RocksDbStorageEngineConfiguration dbStorageEngineConfiguration = ignite.clusterConfiguration()
+                .getConfiguration(RocksDbStorageEngineConfiguration.KEY);
+        int defaultValue = dbStorageEngineConfiguration.flushDelayMillis().value();
+        dbStorageEngineConfiguration.flushDelayMillis().update(defaultValue);
```

Comment:
Please "join" this future before stopping the node

Reply:
But, please either do it with a timeout, or use "assertThar(fut, willCompleteSuccessfully())"